src/js/services/fingerprintService.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+angular.module('copayApp.services').factory('fingerprintService', function(gettextCatalog) {
+  var root = {};
+
+  var requestTouchId = function(cb) {
+    try {
+      window.plugins.touchid.verifyFingerprint(
+        gettextCatalog.getString('Scan your fingerprint please'),
+        function(msg) {
+          $log.debug('Touch ID OK');
+          return cb();
+        },
+        function(msg) {
+          $log.debug('Touch ID Failed:' + JSON.stringify(msg));
+          return cb(gettextCatalog.getString('Touch ID Failed') + ': ' + msg.localizedDescription);
+        }
+      );
+    } catch (e) {
+      $log.debug('Touch ID Failed:' + JSON.stringify(e));
+      return cb(gettextCatalog.getString('Touch ID Failed'));
+    };
+  };
+
+  root.check = function(client, config, cb) {
```

Comment:
If we agree that configService is OK to be shared in services, we should not pass `config`